gateway/src/apicast/policy/standalone/standalone.lua

```diff
@@ -217,11 +219,31 @@ end
 do
   local External_mt = { __index = External }
 
+  local UpstreamPolicy = Policy.new('Standalone Upstream Policy')
+  local UpstreamPolicy_new = UpstreamPolicy.new
+
+  function UpstreamPolicy.new(upstream)
```

Comment:
Would it make sense to use the exiting .new()? Couldn't we pass the config as the exiting constructor expects it? I guess that a rule that matches everything would work.

Reply:
True. Will address it.